test/losses/test_ssim.py

```diff
@@ -171,19 +181,30 @@ def test_module(self, device, dtype):
         op = kornia.losses.ssim3d_loss
         op_module = kornia.losses.SSIM3DLoss(*args[2:])
 
-        assert_close(op(*args), op_module(*args[:2]))
+        self.assert_close(op(*args), op_module(*args[:2]))
 
-    def test_gradcheck(self, device, dtype):
+    def test_gradcheck(self, device):
         # input data
-        window_size = 3
-        img1 = torch.rand(1, 1, 5, 4, 3, device=device, dtype=dtype)
-        img2 = torch.rand(1, 1, 5, 4, 3, device=device, dtype=dtype)
+        img = torch.rand(1, 1, 5, 4, 3, device=device)
 
         # evaluate function gradient
-        img1 = utils.tensor_to_gradcheck_var(img1)  # to var
-        img2 = utils.tensor_to_gradcheck_var(img2)  # to var
+        img = utils.tensor_to_gradcheck_var(img)  # to var
```

Comment:
As future feature,  maybe tensor to gradcheck should be also in the base tester

Reply:
yep, and maybe we can apply to all tensors on the inputs of `self.gradcheck`

Reply:
Agree